pkg/metrics/metrics.go

```diff
@@ -0,0 +1,67 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+var log = logf.Log.WithName("registration_metrics")
+var Reg *prometheus.Registry
+
+const (
+	ResponseMetricLabelRouted   = "Routed"
```

Comment:
don't forget to either rename this or add a new label to cover successful responses for `RegServProxyResponseHistogramVec` metric

Reply:
removed this label